src/proto/parameters.proto

```diff
@@ -64,26 +67,169 @@ message AttackerTacticConfig
+    // rewards completely. This makes the agent myopic, focusing solely on short-term
+    // gains. A factor closer to 1 makes the agent consider future rewards more
+    // significantly, encouraging it to evaluate long-term benefits.
+    required double discount_factor = 4
+        [default = 0.9, (bounds).min_double_value = 0.0, (bounds).max_double_value = 1.0];
+
+    // Controls the epsilon parameter if using an epsilon-greedy selection strategy.
+    // The epsilon determines the exploration-exploitation balance; higher epsilons lead
+    // to more exploration of the action space, while lower epsilons favor exploitation of
+    // the best-known actions.
+    required double action_selection_epsilon = 5 [
+        default                   = 0.3,
+        (bounds).min_double_value = 0.0,
+        (bounds).max_double_value = 10.0
+    ];
+
+    // Controls the temperature parameter if using a softmax action selection strategy.
+    // Temperature determines the exploration-exploitation balance; higher temperatures
+    // lead to more exploration of the action space, while lower temperatures favor
+    // exploitation of the best-known actions.
+    required double action_selection_temperature = 6 [
+        default                   = 0.3,
+        (bounds).min_double_value = 0.0,
+        (bounds).max_double_value = 10.0
+    ];
+
+    // Reward function parameters
+    required AttackerRewardFunctionConfig reward_function_config = 7;
 }
 
-message CornerKickPlayConfig
+message AttackerRewardFunctionConfig
 {
-    // The maximum time that we will wait before committing to a pass
-    required double max_time_commit_to_pass_seconds = 1
-        [default = 3.0, (bounds).min_double_value = 0.0, (bounds).max_double_value = 5.0];
+    // Reward for a goal scored by the friendly team
+    required double friendly_team_scoring_reward = 1 [
+        default                   = 0.8,
+        (bounds).min_double_value = -1.0,
+        (bounds).max_double_value = 1.0
+    ];
+
+    // Penalty for a goal scored by the enemy team
+    required double enemy_team_scoring_penalty = 2 [
+        default                   = -0.8,
+        (bounds).min_double_value = -1.0,
+        (bounds).max_double_value = 1.0
+    ];
+
+    // Reward for getting the ball near the enemy goal (i.e. in the enemy defense area)
+    required double ball_near_enemy_goal_reward = 3 [
+        default                   = 0.4,
+        (bounds).min_double_value = -1.0,
+        (bounds).max_double_value = 1.0
+    ];
+
+    // Reward for keeping friendly possession
+    required double kept_possession_reward = 4 [
+        default                   = 0.2,
+        (bounds).min_double_value = -1.0,
+        (bounds).max_double_value = 1.0
+    ];
+
+    // Penalty for losing friendly possession
+    required double lost_possession_penalty = 5 [
+        default                   = 0.2,
+        (bounds).min_double_value = -1.0,
+        (bounds).max_double_value = 1.0
```

Comment:
```suggestion
        (bounds).max_double_value = 0.0
```

Reply:
I just realized we are actually rewarding +0.2 for losing possession, oops

Reply:
done